### PR TITLE
[13.0][MIG] account_multicurrency_revaluation: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT20489

### CODE REVIEW

File: account_multicurrency_revaluation/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="res.groups" id="group_revaluation_additional">
+        <field name="name">Additional provisioning entries posting</field>
+        <field name="category_id" ref="base.module_category_accounting_and_finance"/>
```
Comment (pedrobaeza) [inline account_multicurrency_revaluation/security/security.xml]: This category doesn't exist

**Comment by gurneyalex** [comment]
@ernestotejeda can you check the red Travis build?

**Comment by pedrobaeza** [comment]
@gurneyalex @ernestotejeda green!

**Review by pedrobaeza — APPROVED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza changes done

**Comment by Tonow-c2c** [comment]
@ernestotejeda can you resolve the conflicts ?

**Comment by ernestotejeda** [comment]
@Tonow-c2c , conflicts resolved

**Review by alan196 — APPROVED** [review]
Functional Review 👍

**Comment by alexey-pelykh** [comment]
@pedrobaeza I guess this somehow slipped through? https://github.com/OCA/account-closing/issues/126#issuecomment-749917453 Or there's no such issue in v13?

**Comment by pedrobaeza** [comment]
I don't know. We only migrated it, and if there's no test check about that, we can't figure it out.

**Pull request merged**